packages/suite-desktop-core/e2e/tests/onboarding/t1b1/t1b1-recovery-advanced.test.ts

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from '../../../support/fixtures';
+
+test.describe('Onboarding - recover wallet T1B1', { tag: ['@group=device-management'] }, () => {
+    test.use({
+        emulatorStartConf: { model: 'T1B1', version: '1-latest', wipe: true },
+        setupEmulator: false,
+    });
+
+    test.beforeEach(async ({ onboardingPage }) => {
+        await onboardingPage.disableFirmwareHashCheck();
+    });
+
+    test('Incomplete run of advanced recovery', async ({
+        onboardingPage,
+        analyticsPage,
+        devicePrompt,
+        page,
+        trezorUserEnvLink,
+    }) => {
+        // Navigate through onboarding steps
+        await analyticsPage.passThroughAnalytics();
+        await onboardingPage.firmwareContinueButton.click();
+        await onboardingPage.recoverWalletButton.click();
+
+        // Select advanced recovery
+        await page.getByTestId('@recover/select-count/24').click();
```

Comment:
Any reason not to use the fixture RecoverActions and the method selectWordCount that you have created?